web/styles/zulip.css

```diff
@@ -834,8 +834,8 @@ p.n-margin {
 }
 
 .app {
-    width: 100%;
-    height: 100%;
+    min-width: 100%;
+    min-height: 100%;
```

Comment:
What's the story with this change? Any CSS change to this part of the CSS is potentially dangerous.

Reply:
The reason for the `min-height` in particular is due to the current bug, where if we try to drag and drop at the end of a stream, we don't get the expected behavior of uploading. This is because even though we have `min-height: 100%` set in `.app-main` we contradict it with `height: 100%` in `.app` which doesn't account for the scroll behavior present in zulip.

We can get rid of the `min-width` change as it does not lead to any change. But this might be a good property to have given that we don't use the `.app` class outside of setting `pointer-events` and the drag/drop behavior. The fluid width layout is also handled by the `.app-main` class which fills the parent in case of the fluid layout, in such a scenario we want to make sure that it does fill up the entire screen.

[Screencast from 2023-11-09 05-40-28.webm](https://github.com/zulip/zulip/assets/82862779/356592f7-7e80-4fa9-bbf9-e8399ddd4994)

Reply:
I have also mentioned a short reasoning in 09a7fd358eb9504c91a6423d2fb5123abdd111fb, but if you suggest I could definitely elaborate on it, thanks!

Reply:
@sayamsamal's reasoning here seems sound to me. But I think some serious burn-in is required to verify that this change doesn't bring about any unexpected side effects.